fs.go

```diff
@@ -402,10 +413,33 @@ type fsFile struct {
 	t            time.Time
 	readersCount int
 
+	bufferPool   sync.Pool
 	bigFiles     []*bigFileReader
 	bigFilesLock sync.Mutex
 }
 
+// NewBufferReader returns io.Reader of cache content of fsFile.
+func (ff *fsFile) NewBufferReader() (io.Reader, error) {
+	if ff.content == nil {
+		return nil, errors.New("this file does not have any content")
+	}
+
+	b := ff.bufferPool.Get()
+	if b == nil {
+		b = bytes.NewBuffer(ff.content)
```

Comment:
Since you only need an `io.Reader` using `bytes.NewReader` is much better.

Reply:
Oh, you are right. I did not see NewReader function.